packages/audiofileplayer/android/src/main/java/com/google/flutter/plugins/audiofileplayer/AudiofileplayerService.java

```diff
@@ -40,6 +42,7 @@
   private MediaMetadataCompat metadata;
   private List<NotificationCompat.Action> notificationActions;
   private int[] compactNotificationActionIndices;
+  private int smallIconId;
```

Comment:
The reason is to avoid multiple calls to something that could be executed just once, during the initialization of the service, since it won't change between notifications.

I will change that if you prefer the other way, no problem.